api/src/main/java/gov/cms/ab2d/api/controller/BulkDataAccessAPI.java

```diff
@@ -50,9 +68,17 @@
                     allowableValues = ALLOWABLE_OUTPUT_FORMATS, defaultValue = "application/fhir" +
                     "+ndjson"
             )
-            @RequestParam(required = false, name = "_outputFormat") String outputFormat) throws IOException {
-        //String encoded = FHIRUtil.outcomeToJSON(FHIRUtil.getSuccessfulOutcome("OK"));
-        return new ResponseEntity<>(null, null,
+            @RequestParam(required = false, name = "_outputFormat") String outputFormat) {
+        //activeRequestCheck();
+
+        Job job = jobService.createJob(resourceTypes, ServletUriComponentsBuilder.fromCurrentRequest().toUriString());
+
+        String statusURL = ServletUriComponentsBuilder.fromCurrentRequestUri().replacePath
+                (String.format(API_PREFIX + "/status/%s", job.getJobID())).toUriString();
```

Comment:
Status URL isn't correct (it _was_ in the PDP coding challenge): status check is at `/Job/{jobId}/$status`